frontend/.eslintrc

```diff
@@ -7,6 +7,6 @@
     "no-unused-variable": "off",
     "no-restricted-globals": ["error", "event"],
     "no-undef": "off",
-    "import/no-default-export": "error"
+    "import/no-default-export": "off"
```

Comment:
Why  we need to set this to off ?

Reply:
The default export was used when exporting i18n.